<launch>
  <!-- "Outdoors" session -->
  <arg name="teach_outdoors"  value="/home/helio/Roboken/Data/Straight/2014-10-21-yaw-02-00/video.mpg" />
  <arg name="replay_outdoors" value="/home/helio/Roboken/Data/Straight/2014-10-21-yaw-02--2/video.mpg" />
  <arg name="replay_outdoors_030" value="/home/helio/Roboken/Data/Straight/2014-11-14-yaw-01-00/video.mpg" />
  <arg name="replay_outdoors_045" value="/home/helio/Roboken/Data/Straight/2014-11-14-yaw-02-00/video.mpg" />
  <arg name="replay_outdoors_060" value="/home/helio/Roboken/Data/Straight/2014-11-14-yaw-03-00/video.mpg" />

  <!-- "Straight" session -->
  <arg name="teach_straight"  value="/home/helio/Roboken/Data/Straight/2014-10-09-yaw-01-00/video.mpg" />
  <arg name="replay_straight" value="/home/helio/Roboken/Data/Straight/2014-10-09-yaw-02-00/video.mpg" />

  <!-- "Turn left" session -->
  <arg name="teach_turn_left"  value="/home/helio/Roboken/Data/Straight/2014-12-16-yaw-02-00/video.mpg" />
  <arg name="replay_turn_left" value="/home/helio/Roboken/Data/Straight/2014-12-16-yaw-04-00/video.mpg" />

  <!-- "Turn right" session -->
  <arg name="teach_turn_right"  value="/home/helio/Roboken/Data/Straight/2014-12-16-yaw-01-00/video.mpg" />
  <arg name="replay_turn_right" value="/home/helio/Roboken/Data/Straight/2014-12-16-yaw-03-00/video.mpg" />

  <node pkg="dejavu" type="experiment_differences" name="experiment_differences" required="true" output="screen">
<!--     <param name="teach_images"  value="100" /> -->
    <param name="replay_leak"  value="0.98" />
    <param name="replay_padding"  value="12" />
    <param name="teach_padding"  value="30" />
    <param name="path_teach"  value="$(arg replay_outdoors_030)" />
    <param name="path_replay" value="$(arg replay_outdoors_060)" />
  </node>
</launch>
